bin/elastic-server.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -xe
+manage.sh onStart #|| exit $?
```

Comment:
Did some research and unfortunately this has to be a separate wrapper as the coprocess doesn't start until immediately after preStart. We need the coprocess (consul agent) to start prior to onStart. onStart's job is there to check whether it has joined the consul cluster at which point we can verify if other services exist. If we did this in preStart, the consul agent won't be running and will assume every container should become the master.

> Coprocesses are started immediately following the exit of the preStart hook, before polling for health or backend hooks, and before the main application starts. - [Containerpilot Docs](https://www.joyent.com/containerpilot/docs/coprocesses)

I think this could be confusing for developers like me who are just getting started with containerpilot and consul agent running as a coprocess. Is there a reason you guys use `CONSUL_AGENT=1` in most of the examples. What is the purpose of making that optional? Meaning, why wouldn't you always use consul agent on each container?